_config.yml

```diff
@@ -58,3 +57,15 @@ defaults:
       front_end_url: https://ecom.externalintegration.payex.com
       consumer_ssn_se: 199710202392
       consumer_ssn_no: 26026708248
+
+elasticsearch:
+  #url: "http://localhost"          # Required. Supports auth and SSL: https://user:pass@someurl.com
+                                    # Can also read URLs stored in environment variable named
+                                    # BONSAI_URL and ELASTICSEARCH_URL.
+  number_of_shards: 1               # Optional. Default is 1 primary shard.
+  number_of_replicas: 1             # Optional. Default is 1 replica.
+  index_name: "test-psp-developer"  # Optional. Default is "jekyll".
+  default_type: "post"              # Optional. Default type is "post".
+  environments:                     # Optional. Set environments where Searchyll should run
+    - "production"                  # Default runs on all environment if empty
```

Comment:
Is it possible to provide the `environment` dynamically depending on the branch being built?

Reply:
Not sure.
But this is more in the bane of "Local development is not production, but all CI builds are".

Reply:
Aren't CI builds either `stage` or `production`?